mozdownload/factory.py

```diff
@@ -56,10 +56,6 @@ def __init__(self, scraper_type, **kwargs):
         :param changeset: Changeset of the try build to download.
 
         """
-        # If a URL has been specified use to the direct scraper
-        if kwargs.get('url'):
-            scraper_type = 'direct'
```

Comment:
This has been moved to the CLI, because FactoryScraper should not decide that on its own. Especially not when another type of scraper has been specified. URL should simply be ignored in this case similar to other options for the other scrapers.